README.md

```diff
@@ -22,6 +22,9 @@ Documentation can be found [here](https://adam-language-learner.readthedocs.io/e
     ```
     adam_root: PATH_TO_WORKING_COPY_OF_THIS_REPO
     adam_experiment_root: PATH_OUTSIDE_WORKING_COPY_TO_WRITE_EXPERIMENT_DATA_TO
+   
+    conda_environment: NAME_OF_ANACONDA_ENVIRONMENT
```

Comment:
Should this just say `adam`? As in,

```suggestion
    conda_environment: adam
```

Reply:
I think we can assume in our instructions that the anaconda environment is called "adam".